mesonbuild/backend/ninjabackend.py

```diff
@@ -1518,7 +1518,7 @@ def generate_dynamic_link_rules(self, outfile):
  rspfile_content = $ARGS  {output_args} $in $LINK_ARGS {cross_args} $aliasing
 '''
                 else:
-                    command_template = ' command = {executable} $ARGS {output_args} $in $LINK_ARGS {cross_args} $aliasing\n'
+                    command_template = ' command = {executable} $ARGS {cross_args} {output_args} $in $LINK_ARGS $aliasing\n'
```

Comment:
Since d87744138ac72df23ecce950909ac89c1f0decc7 this code no longer exists to be swapped, and the commit message (and other surrounding commit messages) implies this should no longer be a problem.